gems/sorbet-runtime/.rubocop_todo.yml

```diff
@@ -0,0 +1,934 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2020-09-13 14:00:43 UTC using RuboCop version 0.90.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
```

Comment:
Would you mind making an issue to burn this list down so that someone looking to contribute a small change to the codebase can find out about this work?

Reply:
Sure!